Vagrantfile

```diff
@@ -1,14 +1,16 @@
 Vagrant.configure(2) do |config|
+  config.vm.synced_folder ".", "/mk"
```

Comment:
Yes I can confirm that this leads to not synching the CWD in the `/vagrant` path anymore, but only in the `/mk` path.